lib/matplotlib/sankey.py

```diff
@@ -355,81 +355,84 @@ def add(self, patchlabel='', flows=None, orientations=None, labels='',
-                            Valid values are 1 (from/to the top), 0 (from/to
-                            the left or right), or -1 (from/to the bottom).  If
-                            *orientations* == 0, inputs will break in from the
-                            left and outputs will break away to the right.
-          *labels*          list of specifications of the labels for the flows
-                            Each value may be *None* (no labels), '' (just
-                            label the quantities), or a labeling string.  If a
-                            single value is provided, it will be applied to all
-                            flows.  If an entry is a non-empty string, then the
-                            quantity for the corresponding flow will be shown
-                            below the string.  However, if the *unit* of the
-                            main diagram is None, then quantities are never
-                            shown, regardless of the value of this argument.
-          *trunklength*     length between the bases of the input and output
-                            groups
-          *pathlengths*     list of lengths of the arrows before break-in or
-                            after break-away
-                            If a single value is given, then it will be applied
-                            to the first (inside) paths on the top and bottom,
-                            and the length of all other arrows will be
-                            justified accordingly.  The *pathlengths* are not
-                            applied to the horizontal inputs and outputs.
-          *prior*           index of the prior diagram to which this diagram
-                            should be connected
-          *connect*         a (prior, this) tuple indexing the flow of the
-                            prior diagram and the flow of this diagram which
-                            should be connected
-                            If this is the first diagram or *prior* is *None*,
-                            *connect* will be ignored.
-          *rotation*        angle of rotation of the diagram [deg]
-                            *rotation* is ignored if this diagram is connected
-                            to an existing one (using *prior* and *connect*).
-                            The interpretation of the *orientations* argument
-                            will be rotated accordingly (e.g., if *rotation*
-                            == 90, an *orientations* entry of 1 means to/from
-                            the left).
-          ===============   ===================================================
-
-        Valid kwargs are :meth:`matplotlib.patches.PathPatch` arguments:
+        Parameters
+        ----------
+        patchlabel : str
+            Label to be placed at the center of the diagram.
+            Note that *label* (not *patchlabel*) will be passed to the patch
```

Comment:
Simpler: Note that label (not patchlabel) can be passed as kwarg to create an entry in the legend.